indictrans/rom2ind.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python 
+# -*- coding: utf-8 -*-
+
+#Copyright (C) 2015 Irshad Ahmad Bhat
+
+"""
+Transliteration Tool:
+Roman to Indic transliterator
+"""
+
+import re
+import sys
+import string
+import os.path
+import warnings
+
+import numpy as np
+
+from ._utils import wxilp
+from ._decode import viterbi
+
+warnings.filterwarnings("ignore")
+
+class rom_to_ind():
+    """Transliterates words from Roman to Indic script"""
+
+    def __init__(self, lang): 
+        self.lookup = dict()
+        self.tab = '\x01\x04'
```

Comment:
Just curious tab as in '\t' and space as in ' ' ? or this is something else?. Tab is 0x9 and 0x20 is space as per ASCII table